src/randoop/JunitFileWriter.java

```diff
@@ -195,58 +183,86 @@ public File writeDriverFile(String driverClassName) {
     return junitTestSuites;
   }
   
-  public static File writeDriverFile(File dir, String packageName, String driverClassName,
-      List<String> junitTestSuiteNames) {
-    File file = new File(dir, driverClassName + ".java");
+  /*
+   * writes JUnit4 suite based on list of class names
```

Comment:
Might this be multiple files?
Please write as a complete sentence with a capitalized first letter and final punctuation.
This needs to be a Javadoc comment, which starts with /** rather than just /*.